openapi/kas-fleet-manager.yaml

```diff
@@ -2293,16 +2303,177 @@ components:
         cloud_provider: aws
         region: us-east-1
         multi_az: true
-    EnterpriseClusterWithAddonParametersExample:
+        supported_instance_types:
+          id: standard
+          display_name: Standard
+          supported_billing_models:
+            - id: enterprise
+              ams_resource: rhosak
+              ams_product: RHOSAKCC
+              ams_billing_models:
+              - standard
+          sizes:
+            [
+              {
+                id: x1,
+                display_name: "1",
+                ingress_throughput_per_sec: {"bytes": 31457280},
+                egress_throughput_per_sec: {"bytes": 31457280},
+                total_max_connections: 3000,
+                max_data_retention_size: {"bytes": 1073741800000},
+                max_partitions: 1000,
+                max_data_retention_period: "P14D",
+                max_connection_attempts_per_sec: 100,
+                quota_consumed: 1,
+                quota_type: "RHOSAK",
+                capacity_consumed: 1,
+                max_message_size: { "bytes": 1048576},
+                min_in_sync_replicas: 1,
+                replication_factor: 1,
+                supported_az_modes: [
+                  "single",
```

Comment:
```suggestion
```

Reply:
You could further remove this if we want to be precise as standard instance can only be deployed on multi_az clusters and are thus multi_az.

Reply:
I will change it :+1:

Reply:
Updated

Reply:
thanks